Pipfile

```diff
@@ -18,6 +18,7 @@ pytest = "==2.8.2"
 "flake8" = "==3.4.1"
 pylint = "==1.8.2"
 pylint-quotes = "==0.1.6"
+python-dateutil = "*"
```

Comment:
Should you specify this to a version like the rest of the dev packages?

Reply:
I don't think we need to for `dateutil`, as we haven't in SR. Will double check with David

Reply:
Scrap that then, if its using the latest version in SR, I'm fine with that being the same on Validator.

Reply:
We should always try and use the latest. We would only ever pin the version if a newer version caused issues with other dependencies.

Reply:
I agree with @dwyeradam - but only because we have dependency locking (piplock).

If we didn't, then I'd always favour pinning to an absolute version. Have had too many headaches in the past due to incompatible upgrades happening at times I _really_ didn't want them to 😄